Assets/Resources/credits_writeup.txt

```diff
@@ -4,67 +4,211 @@
 
 CREDITS & ACKNOWLEDGMENTS
 
```

Comment:
i think we should just delete this file. i can do it after the merge

Reply:
Oh, I replied in the general PR.